libraries/google/bigquery/property/dataset_access.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: false
```

Comment:
I looked through inspec-gcp and there's currently 48 of these property files (51 after this commit).

Do these need to be separate files? It might be easier to parse (and no require logic!) just to inline these directly into the controls. The property files aren't shared between resources (they're namespaced separately), so there's no real gain to having them as separate files that I can see.

Reply:
They don't need to be separate files. I have a personal preference for having a single class in a file, but there is no technical reason they have to be separate. Do you believe it would make it easier to consolidate these within the resource? 

I think it could be overwhelming on some of the larger, more nested resources.

Reply:
I personally think it would be easier to manage the InSpec codebase at scale with just controls, as opposed to controls + property files.

If you think it'll be quick (and you have time), maybe mock something up so we can see what it'll look like?

Reply:
Refactor of nested object structure happening in next PR due to unrelated namespacing issues, can explore there.